devpi_ldap/main.py

```diff
@@ -175,7 +175,7 @@ def _search(self, conn, config, **kw):
             if any(attribute_name in x.get('attributes', {}) for x in conn.response):
                 def extract_search(s):
                     if 'attributes' in s:
-                        return s['attributes'][attribute_name]
+                        return [s['attributes'][attribute_name]]
```

Comment:
Could you add that with tests? I currently have no way to reproduce this with an actual LDAP server, so I can't be sure I get the test data correct.